README.md

```diff
@@ -47,4 +47,5 @@ Make sure you have Homebrew installed
 Your Name - [@your_twitter](https://twitter.com/your_username) - email@example.com
 
 <!-- ACKNOWLEDGEMENTS -->
-## Acknowledgements
\ No newline at end of file
+## Acknowledgements
+<We need to add Project Owner and team info>
```

Comment:
Make sure to remember to fill in all the placeholder values near the end of the project